src/services/ethereumService.js

```diff
@@ -0,0 +1,251 @@
+import { ethers } from 'ethers'
+import { showToastMsg } from './toastService';
+import TiramisuSavingsClub from '../artifacts/contracts/TiramisuSavingsClub.sol/TiramisuSavingsClub.json'
+const TiramisuAddress = '0xFD60a545a2161895Fe1CDF6bf2B83988dC071318';
+const consoleLoggingEnabled = true;
+
+/**************************************** METAMASK WALLET CALLS *************************************************/
+
+let provider;
+let signer;
+let contract;
+let currentWallet;
+let currentChain;
+let callback;
+
+export function isMetaMaskInstalled(){    
+    const { ethereum } = window;
+    return Boolean(ethereum && ethereum.isMetaMask);
+}
+
+export function isWalletConnected(){
+    if(currentWallet === null || currentWallet === undefined) return false;
+    return true;
+}
+
+export function isConnectedToSupportedNetwork(){
+    if(currentChain === null || currentChain === undefined) return true;
+    return true;
+}
+
+export async function getCurrentWalletAddress(){
```

Comment:
We may want to add a comment to this function that it has a precondition (```attemptWalletConnect``` must be called succesfully first)